ui/apps/platform/src/Containers/VulnMgmt/List/Cves/VulnMgmtListCves.js

```diff
@@ -53,6 +54,14 @@ export const defaultCveSort = [
     },
 ];
 
+function parseCveNamesFromIds(cveIds) {
+    const cveNames = cveIds.map((cveId) => {
+        return cveId.split('#')[0] || '';
```

Comment:
An observation, not a change request that `|| ''` seems to be redundant according to quick and dirty test:
| cveId | split |
| :--- | :--- |
| `'a#b'` | `['a', 'b']` |
| `'a#'` | `['a', '']` |
| `'#b'` | `['', 'b']` |
| `'a'` | `['a']` |
| `''` | `['']` |

Reply:
Just a defensive programming habit